src/spec.c

```diff
@@ -1084,6 +1124,21 @@ void *IndexSpec_RdbLoad(RedisModuleIO *rdb, int encver) {
   } else {
     sp->timeout = RedisModule_LoadUnsigned(rdb);
   }
+
+  if (encver >= INDEX_MIN_ALIAS_VERSION) {
+    size_t narr = RedisModule_LoadUnsigned(rdb);
+    for (size_t ii = 0; ii < narr; ++ii) {
+      QueryError status;
+      size_t dummy;
+      char *s = RedisModule_LoadStringBuffer(rdb, &dummy);
+      int rc = IndexAlias_Add(s, sp, 0, &status);
+      if (rc != REDISMODULE_OK) {
+        fprintf(stderr, "[redisearch]: couldn't add alias for %s while loading: %s\n", sp->name,
```

Comment:
How can this happened? and if it does we should not load the rdb, its better to crash here.